ext/spl/tests/spl_object_id.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+SPL: spl_object_id()
+--FILE--
+<?php
+
+var_dump(spl_object_id(new stdClass));
+var_dump(spl_object_id(42));
+var_dump(spl_object_id());
+$a = new stdClass();
+var_dump(spl_object_id(new stdClass) === spl_object_id($a));
+
+?>
+===DONE===
+<?php exit(0); ?>
```

Comment:
Makes sense, removed. It's fairly common in ext/spl/tests, so I thought it was some sort of convention.

This was based on spl_object_hash.phpt, changing that is out of scope.